include/freerdp/server/rdpgfx.h

```diff
@@ -21,6 +21,7 @@
 #define FREERDP_CHANNEL_RDPGFX_SERVER_RDPGFX_H
 
 #include <freerdp/channels/rdpgfx.h>
+#include <freerdp/freerdp.h>
```

Comment:
unnecessary change, better remove.

Reply:
Umm, is it? It doesn't compile without it. There is usage of the `rdpContext` struct without including `freerdp.h`. Surely it's a better solution, rather than relying on including `freerdp.h` before `server/rdpgfx.h` each time